src/main/resources/swagger/api-docs.yaml

```diff
@@ -3160,6 +3174,40 @@ components:
           type: string
           description: user email
       description: the user's details
+    UserResourcesResponse:
+      required:
+        - resourceId
+        - direct
+        - inherited
+        - public
+        - authDomainGroups
+        - missingAuthDomainGroups
+      type: object
+      properties:
+        resourceId:
+          type: string
+          description: The id of the resource
+        direct:
+          description: roles and actions assigned (via membership or public) by a policy directly on this resource
+          allOf:
```

Comment:
using allOf was the only way I could figure out to have both a $ref and description